task_12/src/scss/index.scss

```diff
@@ -0,0 +1,195 @@
+      background-color: #000000;
+      color: #ffffff;
+      border: 1px solid #000000;
+      position: relative;
+      transition: 0.3s;
+      span {
+        height: 54px;
+        line-height: 54px;
+        font-size: 20px;
+        margin: 0 10px;
+        display: block;
+        border-left: 1px solid #ffffff;
+        border-right: 1px solid #ffffff;
+      }
+    }
+    .add-button-wrapper__button:hover{
+      background-color: lighten(#000000, 20);
+      transition: 0.3s;
+    }
+  }
+  &__pizzas-orders {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+    flex-wrap: wrap;
+    align-content: stretch;
+    .pizzas-orders__pizza-card {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-around;
+      align-items: center;
+      width: 300px;
+      height: 300px;
+      border: 1px solid black;
+      margin: 10px 0;
+      .pizza-card__ordered-details {
+        width: 100%;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-around;
+        align-items: center;
+        flex-wrap: wrap;
+        align-content: stretch;
+        .timer {
+          width: 100%;
+          text-align: left;
+          padding-bottom: 5px;
+        }
+        .number {
+          width: 100%;
+          text-align: right;
+          padding-bottom: 5px;
+        }
+        .time-to-finish {
+          width: 100%;
+          text-align: left;
+        }
+        .cost {
+          width: 100%;
+          text-align: right;
+        }
+      }
+      .pizza-card__img {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        & img {
+          margin-top: 20px;
+          width: 100%;
+        }
+      }
+    }
+  }
+}
+
+.footer {
+  padding: 30px 10px;
+  border-top: 1px solid black;
+  background-color: #375812;
+  color: #ffffff;
+  .footer__wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: center;
+  }
+}
+
+@media (max-width: 500px) {
+  .header {
+    &__wrapper {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      padding: 30px 10px;
+      border-bottom: 1px solid black;
+    }
+    &__timer {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      align-items: center;
+      border: 1px solid black;
+      width: 80%;
+      height: 60px;
+    }
+    &__logo {
+      width: 100px;
+    }
+    &__login {
+      width: 100%;
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      align-items: center;
+    }
+    &__login-button {
+      display: flex;
+      flex-direction: row;
+      justify-content: center;
+      align-items: center;
+      border: 1px solid black;
+      width: 80%;
+      height: 60px;
+      cursor: pointer;
+    }
+  }
+}
```

Comment:
Always leave newline character in the end of file.

Reply:
+ done